lib/browserSync.js

```diff
@@ -9,10 +9,10 @@ it.
 
 const browserSync = require('browser-sync').create();
 
-function runBrowserSync ({ port, portProxy }) {
+function runBrowserSync ({ port, portProxy, siteName }) {
   browserSync.init({
     port,
-    proxy: `localhost:${portProxy}`,
+    proxy: `localhost:${portProxy}/content/${siteName}.html`,
```

Comment:
Do we want to always open this link? Is there any chance to standardize behavior with the "open" button after page creation?

Reply:
I think this is already good start, but yes it's likely you would want be able to configure:
- whether or not to open in edit or viewAsPublished
- the start page to open

so basically the same as we implemented in the open action after site creation. Unfortunately we're detached here from the template configuration and cannot reuse it as far as I can tell.

Reply:
we could optionally export additional vars for "download theme sources" as we do already interact with those then we could mimic the same behavior in proxy.

Reply:
thinking more about it, reusing the site template configuration seems like an undesired/hidden behaviour. The `openItem` configuration in the site template can be custom and for example, used to open a second configuration url post site creation. In that case you would not want to reuse such a url in the proxy.

I would rather have a simple way of configuring similar properties in the theme builder (env variable, flag etc.). For now, let's keep this simple and discuss this idea in the review next week.